EDF.py

```diff
@@ -1,3 +1,36 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+""" pyedf is a python package to read from and write EEG data to European Data 
+    Format files. Since EDF is such a widely used format, there exist multiple 
+    Python implementations for reading and writing EDF files. However, most of 
+    these Python modules consist of wrappers around the C-code implementation, 
+    which makes installation more cumbersome and reduces portability. This 
+    implementation is in pure python with limited dependencies on external 
+    packages while having support for Python 2.7 and 3.x.
+
+    Note: the EDF header is represented as a tuple of (meas_info, chan_info)
+        meas_info should have the following: [
+            'record_length', 'file_ver', 'hour', 'subject_id',
+            'recording_id', 'n_records', 'month', 'subtype',
+            'second', 'nchan', 'data_size', 'data_offset', 
+            'lowpass', 'year', 'highpass', 'day', 'minute']
+        chan_info should have the following: [
+            'physical_min', 'transducers', 'physical_max', 
+            'digital_max', 'ch_names', 'n_samps', 'units', 'digital_min']
+    
+    The EDF standard was introduced in 1992. The extension of EDF with 
+    annotations was first described in 1998 and more formalized with the EDF+ 
+    standard that was published in 2003. To learn more about both standards and 
+    implementation details, check out https://www.edfplus.info/index.html
+"""
+__version__ = "0.4.0"
```

Comment:
Where did the version number come from?

Reply:
This one I wanted a discussion about. I don't see any versioning in original library. SO I gave it one by  counting the git commits in original one. If the library is to be without versioning, then I think removal is better.

Reply:
I would just go with `0.2.0` (allowing the previous version to be an implicit `0.1.0`) and follow [SemVer](https://semver.org/).